Exercises/NoModules/Chapter 07/Soln7_07.cpp

```diff
@@ -0,0 +1,43 @@
+// Exercise 7-7
+// Practice string concatenation, looping over strings, and stoi()-like functions
```

Comment:
I had to type # twice to stop input:

Enter a sequence of numbers terminated by #:
`1 2 3 5     2#`
`#`
`The sum of the numbers you entered is: 13`

Reply:
That happens if you do not add a space between the last number and the #. For some reason I always did that differently. Needs fixing, yes...